tests/rt.conf

```diff
@@ -2,7 +2,7 @@
 # S2S tests                                                                                                                                                                    #
 ###################################################################################################################################################################################
 
-COMPILE | -DAPP=S2SW -DUFS_GOCART=ON -DCCPP_SUITES=FV3_GFS_v16_coupled_nsstNoahmpUGWPv1,FV3_GFS_v16_coupled_p7_rrtmgp,FV3_GFS_v17_coupled_p8      | - wcoss_cray                            | fv3 |
+COMPILE | -DAPP=S2SW -DCCPP_SUITES=FV3_GFS_v16_coupled_nsstNoahmpUGWPv1,FV3_GFS_v16_coupled_p7_rrtmgp,FV3_GFS_v17_coupled_p8      | - wcoss_cray                            | fv3 |
```

Comment:
Can we use S2SWA for fv3+mom6+cice6+ww3+gocart application as we may want to keep s2sw for certain application such as GEFS ensemble members

Reply:
Done.